### ISSUE TITLE
MaM, Special Mobs, Elemental Creepers support

### ISSUE THREAD

**Issue by Adaptivity** [ganymedes01/Headcrumbs#32]
[MaM](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2157350-1-7-2-1-7-10-v1-0-2-myths-and-monsters-mod-new), [Special Mobs](http://minecraft.curseforge.com/mc-mods/59968-special-mobs), [Elemental Creepers](http://minecraft.curseforge.com/mc-mods/71270-elemental-creepers). What do you think about those?

**Comment by ganymedes01** [comment]
I'll look into them. But for now I need a break from adding heads XD

Elemental Creepers should be quite easy so I think I'll do those :3

**Comment by Adaptivity** [comment]
Yeah, it's ok. I guess the main problem is textures, right? :)

**Comment by ganymedes01** [comment]
Not really, the textures Headcrumbs pulls from the mods so I don't really need to do anything about that.
The real problem are the models! There is not standard as to what's a "head" and what isn't... so I need to manually go through each model and copy over the part the represents the head, tweak values sometimes and etc... which is easy, but when I have to do that to hundreds of mobs it gets incredibly tedious.

So for mods that don't have different models (like Elemental Creepers where every head is just a simple cube) I can easily add support for it (as I just did), but mods with intricate and different models (like Grimoire of Gaia) it takes forever.

**Comment by Adaptivity** [comment]
Understood. Take your time :smile:

**Comment by DrZed** [comment]
Wait github has check boxes for issues?

**Comment by ganymedes01** [comment]
lol
Just type: -_[_] (replace _ with spaces)
